libraries/package-deps-hash/src/getPackageDeps.ts

```diff
@@ -70,18 +71,34 @@ export function parseGitStatus(output: string, packagePath: string): Map<string,
        *   - '??' == untracked
        *   - 'R' == rename
        *   - 'RM' == rename with modifications
-       * filenames == path to the file, or files in the case of files that have been renamed
+       * filenames == path to the file, or files in the case of files that have been renamed. When filename characters
+       * are escaped, filenames will be surrounded by double-quotes. Some systems allow double-quotes in the
```

Comment:
Did you test this on Windows/Mac/Linux?  Any shell escaping may be different.

Reply:
I tested on Linux. Escaping there was done with a backslash, with the entirety of the string surrounded by double-quotes. I've also seen double-quotes on Windows, but can't seem to remember the scenario that led to them being used.